pandas/core/arrays/sparse.py

```diff
@@ -2025,3 +2056,126 @@ def to_coo(self, row_levels=(0, ), column_levels=(1, ), sort_labels=False):
                                                  column_levels,
                                                  sort_labels=sort_labels)
         return A, rows, columns
+
+    def to_dense(self):
+        from pandas import Series
+        return Series(self._parent.array.to_dense(),
+                      index=self._parent.index,
+                      name=self._parent.name)
+
+
+class SparseFrameAccessor(PandasDelegate):
+
+    def __init__(self, data=None):
```

Comment:
Had to duplicate this `__init__` method since PandasDelegate is a descriptor. Actually, ... let me try out

1. Making a BaseAccessor (that is not a PandasDelegate)
2. Multiple Inheritence from BaseAccessor and PandasDelegate